docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+
+### Session kinds
+
+Require and allow policies have to specify which sessions they apply to. Valid options are `ssh` and `k8s`.
+
+### Filters
+
+A filter expression or predicate is a string statement used to define logic based on a set of input variables.
+These follow common programming language syntax and allow for more detailed control.
+`user.name == "adam" || contains(user.traits, "view")` is a simple example of a filter expression
+that evaluates to true if the user is Adam or if the user has the trait `view`.
+
+Require policies can specify which users they consider as valid with a filter expression. They work like
+other filters used within Teleport such with access requests. The filter context has a `user` object defined
+which the set fields `roles`, `traits` and `name`.
+
+### Backwards compatibility with Server Access
+
+Previously, Server Access did not include controls over which users can join a session.
+To work around this, RBAC rules are ignored for users that only have V4 roles (`version: v4` in the role specification).
+New roles are created as V5. V4 roles are upgraded when they are modified in the UI.
+If a user has any attached V5 roles, the new RBAC access checks will be enforced.
+
+### Example require policy
+
+The policy below specifies that a given role
+must have a minimum of two users with the role `auditor` and the mode `moderator` present in the session
+to start it. The policy applies to SSH and Kubernetes sessions only.
+
+When a user with this require policy starts a session, it will be pending
+until the policy is fulfilled.
+
+```yaml
+kind: role
+metadata:
+  name: prod-access
+spec:
+  allow:
+    require_session_join:
+      - name: Auditor oversight
+        filter: 'contains(user.roles, "auditor")'
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator']
+```
+
+### Example allow policy
+
+The following allow policy attaches to the role `auditor` and allows one to join
+SSH and Kubernetes sessions started by a user with the role `prod_access` as a moderator or observer.
+
+```yaml
+kind: role
+metadata:
+  name: auditor
+spec:
+  allow:
+    join_sessions:
+      - name: Auditor oversight
+        roles : ['prod-access']
+        kinds: ['k8s', 'ssh']
+        modes: ['moderator', 'observer']
+```
+
+## MFA-based presence
+
+When `mfa_required` is set to `true` via role or cluster settings, Teleport enforces
+MFA-based presence checks for moderators.
+This requires that all moderators wishing to join have a configured U2F or WebAuthn MFA token.
+
+Every 30 seconds, Teleport will issue a prompt to the user in the terminal, asking them
+to press their MFA token in the next 15 seconds. This will happen continously during the session
+and exists so that moderators are always present and watching a given session.
+
+If no MFA input is received within 60 seconds, the user is kicked
+from the session which may pause it, if RBAC policies are no longer met.
+
+### Participant Modes
```

Comment:
Should we make this an H2-level section since it doesn't have to do with configuring RBAC resources?

Reply:
I've liberally decided to convert most of the headers to H2, maybe It'd make sense to group Session kinds, Filters, example policies and MFA-based presence under a H2 named RBAC configuration and make them H3, what do you think?

Reply:
I've reworked headings now, take a look